content/css/concepts/animations/terms/animation-direction/animation-direction.md

```diff
@@ -84,3 +84,6 @@ div {
   }
 }
 ```
+![Example 1 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example1.gif)
+
+![Example 2 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example2.gif)
```

Comment:
Can we give better names to these tags? As if this gif doesn't load then this text will show.